hack/scripts/direct_lvm.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# WARNING: THIS SCRIPT HAS MUTLIPLE PURPOSES.
+# TAKE CARE WHEN EDITING.
+
+# This is a scripted version of https://docs.docker.com/storage/storagedriver/device-mapper-driver/#configure-direct-lvm-mode-manually
+
+set -ex
+
+if [[ $(id -u) != 0 ]]; then
+  echo "Run this script as root..." >&2
+  exit 1
+fi
+
+DISK_NAME="${1:-sdb}"
+DISK_PATH="/dev/$DISK_NAME"
+POOL_NAME="${1:-flintlock}"
```

Comment:
This is using the same arg (i.e. 1) as the disk name, so the volume group ends up being called the same as the disk and not flintlock.

Reply:
Guessing it should be `POOL_NAME="${2:-flintlock}"`

Reply:
ah yes excellent catch thank you!